torch/onnx/symbolic_opset9.py

```diff
@@ -835,24 +835,49 @@ def _prepare_onnx_paddings(dim, pad):
     paddings = paddings[-2::-2] + paddings[-1::-2]
     return paddings
 
+def _convert_padding_node(padding):
+    if sym_help._is_value(padding):
```

Comment:
My understanding is that padding arg in reflecion_pad and replication_pad ops is always an array of ints.
Could padding be a single int?
Also, can you use _maybe_get_const() function from symbolic_helper to parse 'is'?
If the output is still a torch._C.Value, then throw the exception?

Reply:
You are correct. The padding arg has to be an array of ints, which can be represented either in an `onnx::Constant` node containing a tensor value, or a `prim::ListConstruct` node.

---
In reply to: [441648471](https://github.com/pytorch/pytorch/pull/39651#discussion_r441648471) [](ancestors = 441648471)

Reply:
Sorry I meant sym_helper._get_const here.